capture/noworkflow/now/vis/static/mydataflow.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash 
```

Comment:
Is this file used?

Reply:
nope, @Daniel-02 forgot to delete it after changing how noworkflow is called in the backend